src/index.html

```diff
@@ -137,18 +137,23 @@ <h1 id="video" class="video-title">Teaser</h1>
         <iframe id="promo-video" class="center-block" src="https://www.youtube.com/embed/voLw_IpG2l0" frameborder="0" allowfullscreen></iframe>
 
         <h1 id="sponsors" class="sponsors-title">Sponsors</h1>
-        <div class="row sponsor-images">
+        	<div class="row sponsor-images">
 					<div class="peta col-md-12">
-              <a href="https://www.imaginbank.com/" target="_blank"><img src="assets/img/imagin-bank-300.png"></a>
-          </div>
+              			<a href="https://www.imaginbank.com/" target="_blank"><img src="assets/img/imagin-bank-300.png"></a>
```

Comment:
Please avoid this @AlbertSuarez otherwise it will be impossible to check who introduced what changed when

Reply:
Okay cool! Thanks :smile: